### PR TITLE
[DOCS] Document rec to avoid scheduling IM rules to run faster than an hour

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses https://github.com/elastic/security-docs/issues/1465.

Preview [here](https://security-docs_1487.docs-preview.app.elstc.co/guide/en/security/master/tuning-detection-signals.html#tune-indicator-rules).

### CODE REVIEW

**Review by joepeeples — APPROVED** [review]
Suggested a possible tweak, LGTM!

File: docs/detections/prebuilt-rules/tune-rule-signals.asciidoc [hunk 1]
```diff
@@ -153,6 +153,7 @@ Take the following steps to tune indicator match rules:
 
 * Specify a detailed query as part of the indicator index query. Results of the indicator index query are used by the detection engine to query the indices specified in your rule definition's index pattern. Using no query or the wildcard `***` query may result in your rule executing very large queries.
 * Limit your rule's additional look-back time to as short a duration as possible, and no more than 24 hours.
+* Avoid cluster performance issues by scheduling your rule to run in one-hour intervals, or higher. For example, avoid scheduling an indicator match rule to check for indicators every five minutes; do schedule it to run every hour.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/prebuilt-rules/tune-rule-signals.asciidoc:156]: This is better!

**Review by benironside — APPROVED** [review]

**Review by ecezalp — APPROVED** [review]
LGTM

File: docs/detections/prebuilt-rules/tune-rule-signals.asciidoc [hunk 2]
```diff
@@ -153,6 +153,7 @@ Take the following steps to tune indicator match rules:
 
 * Specify a detailed query as part of the indicator index query. Results of the indicator index query are used by the detection engine to query the indices specified in your rule definition's index pattern. Using no query or the wildcard `***` query may result in your rule executing very large queries.
 * Limit your rule's additional look-back time to as short a duration as possible, and no more than 24 hours.
+* Avoid cluster performance issues by scheduling your rule to run in one-hour intervals or longer. For example, avoid scheduling an indicator match rule to check for indicators every five minutes.
 
 NOTE: {es-sec} provides limited support for indicator match rules. See <<support-indicator-rules, support limitations>> for more information.
```
Comment (jmikell821) [inline docs/detections/prebuilt-rules/tune-rule-signals.asciidoc:158]: Just pointing out these things as I see them. Your choice of replacement for "see". 
```suggestion
NOTE: {elastic-sec} provides limited support for indicator match rules. Visit <<support-indicator-rules, support limitations>> for more information.
```

**Review by jmikell821 — COMMENTED** [review]
One slight comment that was already existing. Looks fine otherwise.

**Pull request merged**